Mathlib/LinearAlgebra/Charpoly/Basic.lean

```diff
@@ -57,6 +57,10 @@ theorem charpoly_monic : f.charpoly.Monic :=
   Matrix.charpoly_monic _
 #align linear_map.charpoly_monic LinearMap.charpoly_monic
 
+open FiniteDimensional in
+lemma charpoly_natDegree [StrongRankCondition R] : natDegree (charpoly f) = finrank R M := by
+  rw [charpoly, Matrix.charpoly_natDegree_eq_dim, finrank_eq_card_chooseBasisIndex]
+
```

Comment:
Is it the case that this can't live next to `[Matrix.charpoly_natDegree_eq_dim](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/Matrix/Charpoly/Coeff.html#Matrix.charpoly_natDegree_eq_dim)` because of imports? If so, please add references to each declaration letting the user know about the exists of the other. If not, please move this next to `Matrix.charpoly_natDegree_eq_dim`.

Potentially, consider adding `eq_finrank_dim` or something for parallelism, but actually I think I prefer your naming.

Reply:
I think there is basically no other cross-refs between `Matrix.charpoly` stuff and `LinearMap.charpoly` stuff. If we want that, I suggest that we do it in a follow-up docs PR.

Reply:
I have no strong feelings here but I'd probably side with @j-loreaux

Reply:
Fair enough. No need to do it in this PR.